src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs

```diff
@@ -529,5 +532,78 @@ public bool OnEditorAction(TextView v, [GeneratedEnum] ImeAction actionId, KeyEv
 			// Otherwise we return true to allow the focus to change correctly.
 			return actionId != ImeAction.ImeNull;
 		}
+
+		partial void OnTextCharacterCasingChangedPartial(DependencyPropertyChangedEventArgs e)
+		{
+			if (_textBoxView == null)
+			{
+				return;
+			}
+
+			var casing = (CharacterCasing)e.NewValue;
+
+			UpdateCasing(casing);
+		}
+
+		private void UpdateCasing(CharacterCasing characterCasing)
+		{
+			var currentFilters = _textBoxView.GetFilters()?.ToList() ?? new List<IInputFilter>();
+
+			//Remove any casing filters before applying new ones.
+			currentFilters.Remove(a => a is InputFilterAllLower || a is InputFilterAllCaps);
+
+			switch (characterCasing)
+			{
+				case CharacterCasing.Lower:
+					var lowerFilter = new List<IInputFilter>(currentFilters)
+										{
+											new InputFilterAllLower()
+										};
+					_textBoxView.SetFilters(lowerFilter.ToArray());
+
+					break;
+				case CharacterCasing.Upper:
+					var upperFilter = new List<IInputFilter>(currentFilters)
+										{
+											new InputFilterAllCaps()
+										};
+					_textBoxView.SetFilters(upperFilter.ToArray());
+
+					break;
+				case CharacterCasing.Normal:
+					break;
+				default:
```

Comment:
Codacy found an issue: [Remove this empty 'default' clause.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6171516)